ui/pages/settings/networks-tab/networks-form/networks-form.js

```diff
@@ -95,17 +94,20 @@ const NetworksForm = ({
     label || (labelKey && t(getNetworkLabelKey(labelKey)));
   const [networkName, setNetworkName] = useState(selectedNetworkName || '');
   const [rpcUrl, setRpcUrl] = useState(selectedNetwork?.rpcUrl || '');
-  const [chainId, setChainId] = useState(selectedNetwork?.chainId || '');
+  const [chainId, setChainId] = useState(selectedNetwork?.caipChainId || ''); // ?
   const [ticker, setTicker] = useState(selectedNetwork?.ticker || '');
   const [blockExplorerUrl, setBlockExplorerUrl] = useState(
     selectedNetwork?.blockExplorerUrl || '',
   );
   const [errors, setErrors] = useState({});
   const [warnings, setWarnings] = useState({});
   const [isSubmitting, setIsSubmitting] = useState(false);
-  const chainIdMatchesFeaturedRPC = FEATURED_RPCS.some(
-    (featuredRpc) => Number(featuredRpc.chainId) === Number(chainId),
-  );
+  const chainIdMatchesFeaturedRPC =
+    chainId &&
```

Comment:
reminder to check on this once `getCaipChainIdFromEthChainId` gets more nailed down